ansible/roles/debops.phpipam/COPYRIGHT

```diff
@@ -0,0 +1,18 @@
+debops.phpipam - Manage phpIPAM installation with Ansible
+
+Copyright (C) 2015-2017 Maciej Delmanowski <drybjed@gmail.com>
+Copyright (C) 2015-2017 DebOps https://debops.org/
+
```

Comment:
@alinalexandru Add yourself here as well. The start date should be 2014 on my line, thanks.

Reply:
Done. Added the company where I work, since they are sponsoring this role

Reply:
Sounds good. You should also update the 2017 year on the DebOps line to 2019, since it's still in the project. Aligning the names like this:
```
Copyright (C) 2015-2019 DebOps https://debops.org/
￼Copyright (C) 2019      Innobyte (Alin Alexandru) https://www.innobyte.com/
```
would make it more readable.

Reply:
Done